channels/sshagent/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# FreeRDP: A Remote Desktop Protocol Implementation
+# FreeRDP cmake build script
+#
+# Copyright 2012 Marc-Andre Moreau <marcandre.moreau@gmail.com>
+#
```

Comment:
I guess it's not the right copyright...

Reply:
I basically copied the file from echo so I can't claim much copyright!  I've added myself, anyway :)